Research/Research/DataModel/Objects/Step/RSDUIStepObject.swift

```diff
@@ -327,7 +327,7 @@ open class RSDUIStepObject : RSDUIActionHandlerObject, Decodable, RSDCopyStep {
         self.title = try container.decodeIfPresent(String.self, forKey: .title) ?? self.title
         
         if let text = try deprecatedContainer.decodeIfPresent(String.self, forKey: .text) {
-            debugPrint("WARNING!!! `text` keyword on a UIStepObject decoding is deprecated and will be deleted in future versions.")
+            // debugPrint("WARNING!!! `text` keyword on a UIStepObject decoding is deprecated and will be deleted in future versions.")
```

Comment:
If we're going to remove this, then let's just ditch it instead of commenting it out

Reply:
I second this!  

![tenor-138451459](https://github.com/NU-MSS/SageResearch/assets/135254543/f715d29f-e5de-4356-af8d-5bc3c30b59dc)